charts/images.yaml

```
@@ -101,7 +101,21 @@ images:
   sourceRepository: github.com/kubernetes/cloud-provider-aws
   repository: registry.k8s.io/provider-aws/cloud-controller-manager
   tag: "v1.26.1"
-  targetVersion: ">= 1.26"
+  targetVersion: "1.26.x"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'protected'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'high'
+      availability_requirement: 'low'
+- name: cloud-controller-manager
+  sourceRepository: github.com/kubernetes/cloud-provider-aws
+  repository: registry.k8s.io/provider-aws/cloud-controller-manager
+  tag: "v1.27.2"
```

There is no such tag yet:
```
% docker pull registry.k8s.io/provider-aws/cloud-controller-manager:v1.27.2
Error response from daemon: manifest for registry.k8s.io/provider-aws/cloud-controller-manager:v1.27.2 not found: manifest unknown: Failed to fetch "v1.27.2"
```

```suggestion
  tag: "v1.27.1"
```